include/click/dpdkdevice.hh

```diff
@@ -1,6 +1,23 @@
 #ifndef CLICK_DPDKDEVICE_HH
 #define CLICK_DPDKDEVICE_HH
 
+//Prevent bug under some configurations (like travis-ci's one) where these macros get undefined
+#ifndef UINT8_MAX
```

Comment:
I agree it's weird, however even with your new commit, it fails (https://travis-ci.org/tbarbette/click/jobs/93178126#L2459 , a version without this define and with your new commit). DPDK seems to undefine (or prevent the inclusion) somehow. I tried a few things, but this awfull hack is the only viable solution I found...